src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/AbstractFilter.php

```diff
@@ -2,12 +2,16 @@
 
 namespace Pim\Bundle\CatalogBundle\Elasticsearch\Filter;
 
-use Akeneo\Component\StorageUtils\Exception\InvalidPropertyException;
 use Pim\Bundle\CatalogBundle\Elasticsearch\SearchQueryBuilder;
-use Pim\Component\Catalog\Model\AttributeInterface;
 use Pim\Component\Catalog\Query\Filter\FilterInterface;
-use Pim\Component\Catalog\Validator\AttributeValidatorHelper;
 
+/**
+ * Basic implementation of a field filter.
```

Comment:
Ok I'll duplicate the logic with the setQueryBuilder + the operators in the AbstractAttributeFilter and AbstractFieldFilter.